Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,9 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {%- if sonata_admin.options['use_icheck'] -%}&nbsp;{%- endif -%}
```

Comment:
@vincenttouzet : here is [the comment you are referring to](https://github.com/sonata-project/SonataAdminBundle/pull/3127#discussion_r54783538). It was about something else, and I personally prefer when the override can be done without CSS, because it seems less fragile (making an override in CSS implies relying on a class that may change for non-obvious reasons). In this case, I can you please follow @ju1ius advice? He's way better at front-end matters than I am.